### PR TITLE
Update lambda ARNs for release v0.25.0

### PR DESCRIPTION
**Pull request by bryan-aguilar**
Update lambda docs with new ARNs. There are some autoformatting changes made in this PR also. I have verified that each page displays correctly locally.

### CODE REVIEW

File: src/content/BlogPosts/blogPosts.yaml [hunk 1]
```diff
@@ -6,6 +6,12 @@ description:
 path: /blog
 
 blogs:
+  - title: "AWS Distro for OpenTelemetry Lambda Layers are now available with ADOT Collector v0.25.0"
+    author: "Bryan Aguilar"
+    date: "11-Jan-2023"
+    body:
+      "ADOT Lambda Layers now distribute ADOT Collector v0.25.0 with new layers supporting AMD64 and ARM64 Lambda functions"
+    link: "/docs/ReleaseBlogs/aws-distro-for-opentelemetry-lambda-layer-v0.24.0"
```
Comment (vasireddy99) [inline src/content/BlogPosts/blogPosts.yaml:14]: link needs to be modified to
`/docs/ReleaseBlogs/aws-distro-for-opentelemetry-lambda-layer-v0.25.0`

File: src/content/Blogs/ReleaseBlogs/aws-distro-for-opentelemetry-lambda-layer-v0.25.0.mdx [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+<SectionSeparator />
+
+### Release Highlights
+
+- Python layer [**aws-otel-python-<amd64|arm64>-ver-1-15-0**](https://aws-otel.github.io/docs/getting-started/lambda/lambda-python) contains OpenTelemetry Python `v1.15.0`
+- Nodejs layer [**aws-otel-nodejs-<amd64|arm64>-ver-1-8-0**](https://aws-otel.github.io/docs/getting-started/lambda/lambda-js) contains OpenTelemetry JavaScript Core `v1.8.0` with AWS Lambda Instrumentation `v0.34.0`
+- Java-Wrapper layer [**aws-otel-java-wrapper-<amd64|arm64>-ver-1-21-0**](https://aws-otel.github.io/docs/getting-started/lambda/lambda-java) contains OpenTelemetry Java `v1.21.0`
+- Java-Agent layer [**aws-otel-java-agent-<amd64|arm64>-ver-1-20-1**](https://aws-otel.github.io/docs/getting-started/lambda/lambda-java-auto-instr) contains AWS Distro for OpenTelemetry Java Instrumentation `v1.20.0`
+- Collector layer **aws-otel-collector-<amd64|arm64>-ver-0-68-0** contains ADOT Collector for Lambda `v0.25.1`. Compatible with [.NET](https://aws-otel.github.io/docs/getting-started/lambda/lambda-dotnet) and [Go](https://aws-otel.github.io/docs/getting-started/lambda/lambda-go) runtimes.
```
Comment (vasireddy99) [inline src/content/Blogs/ReleaseBlogs/aws-distro-for-opentelemetry-lambda-layer-v0.25.0.mdx:39]: should we use ADOT Collector for Lambda `v0.25.0`here although the tag is `v0.25.1`

Comment (Aneurysm9) [reply src/content/Blogs/ReleaseBlogs/aws-distro-for-opentelemetry-lambda-layer-v0.25.0.mdx:39]: It should match whatever is in https://github.com/aws-observability/aws-otel-lambda/blob/main/adot/collector/VERSION as that's the version the collector will report.

**Review by Aneurysm9 — COMMENTED** [review]
Can the formatting changes be pulled out into a separate PR?  It's hard to review this as-is.

**Pull request closed without merging**